backend/.env.development

```diff
@@ -7,4 +7,7 @@ SITE_DOMAIN=orbitar.local
 DEFAULT_TL_LANGUAGE=ru
 
 SENDING_BLUE_API_KEY=
-DEEPL_API_KEY=
\ No newline at end of file
+DEEPL_API_KEY=
+
+MYSQL_USER=orbitar
```

Comment:
I think these are also taken from `.env` for the `NODE_ENV=development` configurations.